tests/integration/mp-graphql/pom.xml

```diff
@@ -127,7 +127,7 @@
                             <goal>jandex</goal>
                         </goals>
                         <!-- phase is 'process-classes by default' -->
-                        <phase>integration-test</phase>
+                        <phase>process-test-resources</phase>
```

Comment:
Just double-checking you meant to change phases

Reply:
Yes. There were intermittent problems because sometimes Maven would run the failsafe plug-in (the integration tests)  (also during `integration-test`) _before_ creating the index. The test did not like that.